BUILD.md

```diff
@@ -72,6 +72,6 @@ From the repo root:
 
     cd debug/test
     make
-    ctest
+    ctest .
```

Comment:
Actually, after the enable_testing() change ``ctest`` works as well as ``ctest .``